dts/bindings/rng/virtio,device4.yaml

```diff
@@ -0,0 +1,8 @@
+# Copyright (c) 2025 TOKITA Hiroshi
+# SPDX-License-Identifier: Apache-2.0
+
+description: VirtIO Entropy device (ID:4)
```

Comment:
AFAICT Virtio should be capitalized Virtio or VIRTIO
Maybe fix across tree in a follow-up